src/main/java/com/gamegoo/gamegoo_v2/content/board/controller/BoardController.java

```diff
@@ -110,5 +111,12 @@ public ApiResponse<String> delete(@PathVariable Long boardId, @AuthMember Member
         return ApiResponse.ok("게시글을 삭제하였습니다.");
     }
 
+    @GetMapping("/my")
+    @Operation(summary = "내가 작성한 게시판 글 목록 조회 API", description = "내가 작성한 게시판 글을 조회하는 API 입니다. 페이지 당 10개의 게시물이 표시됩니다.")
+    @Parameter(name = "pageIdx", description = "조회할 페이지 번호를 입력해주세요.")
+    public ApiResponse<MyBoardResponse> getMyBoardList(@RequestParam(defaultValue = "1") int pageIdx,
```

Comment:
게시판 글 목록 조회 API처럼 @ValidPage 사용해서 BoardService에서는 페이지 값 검증하지 않도록 해도 좋을 것 같습니다..!
```suggestion
            @ValidPage @RequestParam(name = "page") Integer page,
```